src/test/groovy/RangeTest.groovy

```diff
@@ -262,7 +262,24 @@ class RangeTest extends GroovyTestCase {
         assert (0..<upper2).size() == 4
     }
 
-    protected void assertIterate(range, expected) {
+    void testDecimalRanges() {
```

Comment:
The rest of my tests were already merged in by @paulk-asert in https://github.com/apache/groovy/commit/d6252c31ebc2caf98d546af2bdfd45aa53166085